snapcraft/project/_project.py

```diff
@@ -32,13 +32,18 @@ def __init__(
         target_deb_arch=None,
         debug=False,
         snapcraft_yaml_file_path=None,
-        project_dir: str = None
+        project_dir: str = None,
+        work_dir: str = None,
+        is_managed_host: bool = False
     ) -> None:
 
-        if project_dir is None:
-            self.project_dir = os.getcwd()
+        project_dir = os.getcwd()
```

Comment:
Hmm. Why support `project_dir` as an argument to this function at all, then?

Reply:
I will have to check, it might be a left over

Reply:
issue taken care of